core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -329,6 +334,7 @@ object AppType {
       case Galaxy   => FormattedBy.Galaxy
       case Cromwell => FormattedBy.Cromwell
       case Custom   => FormattedBy.Custom
+      case Wds      => FormattedBy.Cromwell
```

Comment:
Creating a new "FormattedBy" enum adds a lot of boiler plate for no functional change. For this reason I'm in favor of keeping this as is, but am open to going back through an renaming the current enum

Reply:
Is this supposed to be `FormattedBy.Cromwell` as well? Or something else?